hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -451,9 +482,8 @@ private Void updateLastTargetQueries(final String tenant, final List<TargetPoll>
 
     /**
      * Sets {@link Target#getLastTargetQuery()} by native SQL in order to avoid
-     * raising opt lock revision as this update is not mission critical and in
-     * fact only written by {@link ControllerManagement}, i.e. the target
-     * itself.
+     * raising opt lock revision as this update is not mission critical and in fact
+     * only written by {@link ControllerManagement}, i.e. the target itself.
```

Comment:
which format is the proper one

Reply:
Using the eclipse code formatter, I am also getting the format that you committed

Reply:
Alright then it should be the case that it wasn't formatted for a long time and by that fine by now.